examples/drain3.ini

```diff
@@ -16,8 +16,9 @@ mask_prefix = <:
 mask_suffix = :>
 
 [DRAIN]
+matched_pattern = JaccardDrain
```

Comment:
Not sure `matched_pattern` is a good name here. Perhaps call it `engine`?

Reply:
That sounds good, I agree.